### PR TITLE
Esas/runtype dependents scenario report

### PR DESCRIPTION
**Pull request by esasova**
Change dashboard config file (src/config/Dashboard.js) :
replace SCENARIO_DASHBOARD_CONFIG with SCENARIO_VIEW_REPORTS_BY_RUNTEMPLATE.
The new config contains all possible dashboards for the project ; every dashboard contains possible filters and other options and is identified with a key that corresponds to currentScenario.data.runTemplateId in Redux. If the same config is used for several runTemplates, it must be declared for each runTemplate (in the defaultScenarioReport variable for example). 

Then, in Scenario view (/src/views/Scenario/Scenario.js), the config is stored in the currentScenarioRunTemplateReport array that helps distinguish new config from old one (for backward compatibility currentScenarioRunTemplateReport must be an array to be compatible with existing SimplePowerBIReportEmbed component(cosmotech/ui library)) : if the old one, the entire config object is assigned to currentScenarioRunTemplateReport, if new one, the SCENARIO_VIEW_REPORTS_BY_RUNTEMPLATE array is filtered for runTemplateId and the result is pushed in currentScenarioViewReport. To avoid app crash when there is no scenario, the first report in the list is declared as defaultPowerBIReport.

The powerbi-client-react actually uses setFilters() and removeFilters() methods to update the configuration while Microsoft recommends the general updateFilters() method : https://docs.microsoft.com/en-us/javascript/api/overview/powerbi/control-report-filters#update-filters
  This causes an undefined error in console nevertheless the config is updated properly. To avoid any confusion or future incompatibility, the embed component is destroyed and rerendered when a scenario changes.

Dev Review rules:
- 3 PR max
- at the second feedback, prefer meet instead of github conversation

Definition of Done:
- [ ] Testing
- [ ] No error logs in web console (except iFrame errors)
- [ ] Create or Update documentation (README file for users/integrators)
- [ ] Identify breaking changes with [conventional commits](https://kapeli.com/cheat_sheets/Conventional_Commits.docset/Contents/Resources/Documents/index)
- [ ] Check the [test page](https://spaceport.cosmotech.com/confluence/pages/viewpage.action?spaceKey=QA&title=Azure+Sample+Webapp) in order to manually validate the feature
- [ ] Clean code (regroup configuration variable, typos, no console.log...)

### CODE REVIEW

**Review by nborde-CSM — APPROVED** [review]
All LGTM :+1:

File: src/views/Scenario/Scenario.js [hunk 1]
```diff
@@ -323,8 +328,9 @@ const Scenario = (props) => {
       </Grid>
       <Card>
         <SimplePowerBIReportEmbed
+          key={currentScenario?.data?.id}
```
Comment (csm-thu) [inline src/views/Scenario/Scenario.js:331]: Do we need to add a key to this component ?

File: src/config/Dashboards.js [hunk 2]
```diff
@@ -13,37 +13,86 @@ import {
...
       panes: {
         filters: {
-          expanded: false,
-          visible: false,
+          expanded: true,
+          visible: true,
         },
       },
     },
-    staticFilters: [new PowerBIReportEmbedMultipleFilter('Bar', 'id', ['MyBar', 'MyBar2'])],
```
[the remark this answers is not in the document]
Comment (csm-thu) [reply src/config/Dashboards.js:32]: Ok, thank you :)

File: src/views/Scenario/Scenario.js (hunk 1 above)
Comment (csm-thu) [reply src/views/Scenario/Scenario.js:331]: Ok, adding a comment to explain why we should keep the `key` might be helpful later :smile:

**Review by csm-thu — APPROVED** [review]
LGTM :+1:

**Pull request merged**